.gitignore

```diff
@@ -18,8 +18,8 @@ MANIFEST
 /*.json
 */cache/*
 !*/cache/.keep
-vsac_wrangler/data/cache/*
```

Comment:
## Close issue when merged
- [ ] @joeflack4 was trouble w/ GitHub being unstable, and I couldn't link #463. Close it after this is merged.